vas2nets/views.py

```diff
@@ -0,0 +1,23 @@
+from datetime import timedelta
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from hellomama_registration import utils
+from .tasks import fetch_voice_data
+
+
+class FetchVoiceDataView(APIView):
+
+    """ FetchVoiceData Interaction
+        POST - starts up the task that pulls voice data from the vas2nets
+        portal
+    """
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, *args, **kwargs):
+        status = 201
+        date = utils.get_today() - timedelta(1)
+        fetch_voice_data.apply_async(args=[date.strftime('%Y-%m-%d')])
+        resp = {"fetch_voice_data_initiated": True}
```

Comment:
Maybe we can return the UUID of the task in the response? In case we ever want to keep track of it?